services/site-scout/launch-worker.sh

```diff
@@ -56,23 +56,42 @@ fi
 update-ec2-status "Setup: fetching build"
 
 # select build
-echo "Build types: ${BUILD_TYPES}"
-BUILD_SELECT_SCRIPT="import random;print(random.choice(str.split('${BUILD_TYPES}')))"
-build="$(python3 -c "$BUILD_SELECT_SCRIPT")"
-# download build
-case $build in
-  asan32)
-    # TEMPORARY workaround for frequent OOMs
-    export ASAN_OPTIONS=malloc_context_size=20:rss_limit_heap_profile=false:max_malloc_fill_size=4096:quarantine_size_mb=64
-    python3 -m fuzzfetch -n build --fuzzing --asan --cpu x86
-    ;;
-  debug32)
-    python3 -m fuzzfetch -n build --fuzzing --debug --cpu x86
-    ;;
-  *)
-    python3 -m fuzzfetch -n build --fuzzing "--$build"
-    ;;
-esac
+if [[ -n "$COVERAGE" ]]; then
+  export COVERAGE_FLAG="--coverage"
+  python3 -m fuzzfetch -n build --fuzzing --coverage
+  export ARTIFACT_ROOT="https://community-tc.services.mozilla.com/api/index/v1/task/project.fuzzing.coverage-revision.latest/artifacts/public"
+  SOURCE_URL="$(resolve-url "$ARTIFACT_ROOT/source.zip")"
+  export SOURCE_URL
+
+  if [[ -z "$REVISION" ]]; then
```

Comment:
Pointless to specify `REVISION` when you've already used `SOURCE_URL` above from latest.